app/experimenter/nimbus-ui/src/components/RichText/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
As in, is there a component that we can use instead of rolling our own? Probably. There's almost certainly one to convert links, but both links and newlines? Not sure.

Reply:
@jaredlockhart while your here, did you see the comment above about backend?